libs/coin-modules/coin-tron/src/bridge/synchronization.ts

```diff
@@ -80,6 +81,7 @@ export const getAccountShape: GetAccountShape<TronAccount> = async (
   const balance = computeBalanceBridge(acc);
 
   const parentTxs = txs.filter(isParentTx);
+  // why don't i use listOperations here? Because two types TronOperation and Operation<TronToken> do not have the same structure
```

Comment:
@sprohaszka-ledger we need to have logic method to output backend raw response and then adapt in bridge / api folder ?

Reply:
The best again is to abstract the backend response. Exposing the backend raw response is not a good idea as it may evolve and therefore have incidence on both `bridge` and `api`.
The `logic`folder will expose its own model to the `bridge` and `api` folder.